script.js

```diff
@@ -1,35 +1,167 @@
-const sideNav = document.getElementById("sideNav")
+const sideNav = document.getElementById('sideNav');
 
 function openNav(open) {
-  sideNav.style.width = "100%";
+  document.getElementById('sideNav').style.width = '100%';
+  sideNav.style.width = '100%';
   return open;
 }
 
+document.getElementById('toggleSpan').onclick(openNav);
+
 function closeNav(close) {
-  sideNav.style.width = "0";
+  document.getElementById('sideNav').style.width = '0';
+  sideNav.style.width = '0';
   return close;
 }
 
-// CONTACT VALIDATION
+document.getElementById('sideNav').onclick(closeNav);
+
+const popup = document.getElementById('popupCard');
```

Comment:
- [ ] In the mobile view, your design doesn't look completely like the Figma design. Kindly refer to the image below and make the necessary changes while referring to the Figma Design 💯 

- Your Design
![F5570B8E-B390-4C8B-8C7E-6C53165FD929](https://user-images.githubusercontent.com/102699958/232234184-9a5167a1-43fd-4898-b8b0-cc2b3cccb03d.jpeg)
- The Figma Design
![0F7DB58B-62F9-404E-8964-B05B5749FE3D_4_5005_c](https://user-images.githubusercontent.com/102699958/232234310-fab1df22-89dd-4877-b56a-6c32fa864a87.jpeg)

- [ ] In the desktop view, there are a few style issues that need to be fixed as well. I have pointed them out in the screenshot below 👍 
- Your Design
![077006AC-9F18-4F9A-9454-F87802328778](https://user-images.githubusercontent.com/102699958/232234852-13b30d77-9c0f-4558-984b-7f5e739d70cd.jpeg)

- The Figma Design
![A17286D4-6AEA-43FA-8F4C-2CC50A76030B_4_5005_c](https://user-images.githubusercontent.com/102699958/232234929-351637cc-5d09-4743-ae84-bc9d0316e39f.jpeg)